crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -1222,16 +1371,29 @@ fn create_c8y_converter() -> (TempTedgeDir, CumulocityConverter<FakeC8YHttpProxy
         .dir("tedge")
         .dir("agent")
         .file("software-list-2011-11-11T11:11:11Z.log");
+    let cfg_dir = TempTedgeDir::new();
+    create_thin_edge_operations(&cfg_dir);
```

Comment:
Doing this here will affect other tests that don't expect these additional `operation` files. I'd rather call this function only in tests that expects these additional directories and files.

Reply:
Agree, but even if I create the operations in the tests where its required, still it changes most of the test which uses the support functions like `create_c8y_converter` or `start_c8y_mapper`

Reply:
Now I refactored the code to create the `operations` in the test itself